src/GovUk.Education.ExploreEducationStatistics.Common/Utils/DbContextUtils.cs

```diff
@@ -19,19 +19,31 @@ public static void UpdateTimestamps(object? sender, EntityEntryEventArgs e)
                 switch (e.Entry.Entity)
                 {
                     case ICreatedTimestamp<DateTime> entityWithCreated:
-                        entityWithCreated.Created = now;
+                        if (entityWithCreated.Created == null)
+                        {
+                            entityWithCreated.Created = now;
+                        }
```

Comment:
Yeah - tbf, it was never a use case before, because in general there's no need to set a CreatedDate manually.  This is really just in place to help us with supporting test data setup!